packages/orbit-components/src/Box/README.md

```diff
@@ -55,7 +55,7 @@ Table below contains all types of the props available in the Box component.
 
 ### Media Queries
 
-When you need to specify some different behaviour of the Box component on different viewport, you can use properties for it.
```

Comment:
is UK variation not correct? 🤔 :D

Reply:
https://orbit.kiwi/content/grammar-and-mechanics/#spelling